CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # 2.0.9 (2023-06-06)
 
+* Add startupProbe (#97)
+
+# 2.0.9 (2023-06-06)
```

Comment:
The `# 2.0.9` heading is in here twice, I think this should be 2.0.10 once

Reply:
yup, sorry. fixed

Reply:
I noticed that too, I fixed it right after.